### ISSUE TITLE
TFselftraining parameters in the paper ?

### ISSUE THREAD

**Issue by Shiro-LK** [yuanli2333/Teacher-free-Knowledge-Distillation#14]
Thanks for sharing the results of the paper.

I have a question about the TeacherFreeSelf Training loss function, written in `my_loss_function.py` .
I see a parameter after the KLDiv with the `T*T` coefficient. I was wondering where does it come from ? I haven't see it mentionned in the paper. Is it normal ?

**Comment by yuanli2333** [comment]
Hi, for the normal KD and self KD, the T\*T is necessary.  There will be 1/(T\*T) in the denominator:
<img width="909" alt="image" src="https://user-images.githubusercontent.com/33076543/85911829-3856c700-b85a-11ea-80e7-58e7480174ea.png">

You can check the details from the original KD paper.
